keyboards/macroflow_original/keyboard.json

```
@@ -0,0 +1,53 @@
+{
+    "manufacturer": "Customacros",
+    "keyboard_name": "Macroflow Original",
+    "maintainer": "Patrickemm",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "oled": true,
+        "encodermap": true
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B3", "B2", "F7"],
+        "rows": ["E6", "B4", "B5"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://github.com/Patrickemm",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x504D"
+    },
+    "layouts": {
+        "LAYOUT_macroflow_2x4": {
```

```suggestion
        "LAYOUT": {
```
And the keymap.c needs updating to reflect that as well.